src/web/mjs/engine/Storage.mjs

```diff
@@ -778,20 +778,25 @@ export default class Storage {
     /**
```

Comment:
Comment can be removed since this sanitise is now no longer based on HakuNeko Legacy